src/openzeppelin/access/ownable/dual_ownable.cairo

```diff
@@ -26,18 +26,16 @@ trait DualCaseOwnableTrait {
 
 impl DualCaseOwnableImpl of DualCaseOwnableTrait {
     fn owner(self: @DualCaseOwnable) -> ContractAddress {
-        (*call_contract_syscall(
-            *self.contract_address, constants::owner_SELECTOR, ArrayTrait::new().span()
-        )
+        (*call_contract_syscall(*self.contract_address, selectors::owner, ArrayTrait::new().span())
             .unwrap_syscall()
             .at(0))
             .try_into()
             .unwrap()
     }
 
     fn transfer_ownership(self: @DualCaseOwnable, new_owner: ContractAddress) {
-        let snake_selector = constants::transfer_ownership_SELECTOR;
-        let camel_selector = constants::transferOwnership_SELECTOR;
+        let snake_selector = selectors::transfer_ownership;
+        let camel_selector = selectors::transferOwnership;
```

Comment:
i don't think we need these bindings anymore, we can just use the selectors directly no?

Reply:
Agreed, good call!